runtime/integration-tests/src/cases/lp/investments.rs

```diff
@@ -156,7 +155,7 @@ mod with_pool_currency {
 	use super::{utils, *};
 	use crate::cases::lp::utils as lp_utils;
 
-	#[test_runtimes([development], ignore = "solidity mismatch")]
+	#[test_runtimes([development])]
```

Comment:
FYI: There is a TODO 10 lines above.

Reply:
Thanks, that's from @mustermeiszer and not related to the LPv2 refactor I have applied.